src/EA.Weee.Web/Areas/AatfReturn/Views/Shared/EditorTemplates/AatfAddressData.cshtml

```diff
@@ -10,41 +10,6 @@
             @Html.Gds().TextBoxFor(m => m.Name, new { @maxlength = CommonMaxFieldLengths.DefaultString })
         </div>
 
-        <!--@Html.Partial("_AddressPartial", Model)-->
-        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.Address1)">
-            @Html.Gds().LabelFor(m => m.Address1)
-            @Html.Gds().ValidationMessageFor(m => m.Address1)
-            @Html.Gds().TextBoxFor(m => m.Address1, new { @class = "govuk-!-width-full", @maxlength = CommonMaxFieldLengths.AddressLine })
-        </div>
-
-        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.Address2)">
-            @Html.Gds().LabelFor(m => m.Address2)
-            @Html.Gds().ValidationMessageFor(m => m.Address2)
-            @Html.Gds().TextBoxFor(m => m.Address2, new { @class = "govuk-!-width-full", @maxlength = CommonMaxFieldLengths.AddressLine })
-        </div>
-
-        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.TownOrCity)">
-            @Html.Gds().LabelFor(m => m.TownOrCity)
-            @Html.Gds().ValidationMessageFor(m => m.TownOrCity)
-            @Html.Gds().TextBoxFor(m => m.TownOrCity, new { @class = "govuk-!-width-full", @maxlength = CommonMaxFieldLengths.AddressLine })
-        </div>
-
-        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.CountyOrRegion)">
-            @Html.Gds().LabelFor(m => m.CountyOrRegion)
-            @Html.Gds().ValidationMessageFor(m => m.CountyOrRegion)
-            @Html.Gds().TextBoxFor(m => m.CountyOrRegion, new { @class = "govuk-!-width-full", @maxlength = CommonMaxFieldLengths.AddressLine })
-        </div>
-
-        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.Postcode)">
-            @Html.Gds().LabelFor(m => m.Postcode)
-            @Html.Gds().ValidationMessageFor(m => m.Postcode)
-            @Html.Gds().TextBoxFor(m => m.Postcode, new { @class = "govuk-!-width-full", @maxlength = CommonMaxFieldLengths.Postcode })
-        </div>
-
-        <div class="govuk-form-group countries @Html.Gds().FormGroupClass(m => m.CountryId)" id="country-dropdown-list">
-            @Html.Gds().LabelFor(m => m.CountryId)
-            @Html.Gds().ValidationMessageFor(m => m.CountryId)
-            @Html.Gds().DropDownListFor(m => m.CountryId, new SelectList(Model.Countries, "Id", "Name"), string.Empty, new { @class = "govuk-!-width-full", autocomplete = "false", aria_labelledby = "country-dropdown-list" })
-        </div>
+        @Html.Partial("_AddressPartial", Model)
```

Comment:
We had issues with the _AddressPartial so had to keep the views having their own fields... You can create a _CountyPrtial that just outputs the country field

Reply:
I've fixed the weird formatting and moved all of the address fields except for the country field back to the editor templates.